docs/source/CartesianConfig.rst

```diff
@@ -570,6 +570,11 @@ Formal definition
       value of <key> in all dicts of the current frame. If a dict lacks
       <key>, it will be created.
 
+   -  A statement of the form <key> ~= <value> sets the value of <key>
+      to <value>, in all dicts of the current frame, but only if <key>
+      does not exist in the dict. If a dict lacks <key>, it will be
+      created.
```

Comment:
how about: `A statement of the form <key> ~= <value> sets the value of <key> to <value> in all dicts of the current frame unless the <key> was already defined. Otherwise the original value is preserved.`

Reply:
Yeah, looks better, thanks!